core/arch/arm/kernel/abort.c

```diff
@@ -108,13 +108,27 @@ static __maybe_unused const char *abort_type_to_str(uint32_t abort_type)
 	return "undef";
 }
 
+static __maybe_unused const char *fault_to_str(uint32_t fault_descr)
+{
+
+	switch (core_mmu_get_fault_type(fault_descr)) {
+	case CORE_MMU_FAULT_ALIGNMENT:
+		return " (alignment fault)";
+	case CORE_MMU_FAULT_TRANSLATION:
+		return " (translation fault)";
+	default:
+		return "";
```

Comment:
Don't we want to output a specific message in case of `CORE_MMU_FAULT_WRITE_PERMISSION` and `CORE_MMU_FAULT_READ_PERMISSION`?